packages/ui/theme-builder/package.json

```diff
@@ -13,6 +13,7 @@
   "dependencies": {
     "@clerk/nextjs": "file:../../nextjs",
     "@clerk/ui": "file:..",
+    "@clerk/localizations": "file:../../localizations",
```

Comment:
Enables pulling in a locale file and providing the the ClerkProvider for testing

```
import { arSA } from '@clerk/localizations';

export default function RootLayout({
  children,
}: Readonly<{
  children: React.ReactNode;
}>) {
  return (
    <ClerkProvider localization={arSA}>
      <html>
        <body>
          <ThemeBuilder>{children}</ThemeBuilder>
        </body>
      </html>
    </ClerkProvider>
  );
}
```